### DISCUSSION TITLE
Система мониторинга и оценки

### DISCUSSION THREAD

**Discussion by ivev7** [totumonline/totum-ru-issues-and-discussions#16] [Сделанные проекты]
Доброго времени суток!

У нас работает система учета клиентов , контактов с ними и оказанных услуг.
Я не программист, первоначально делал систему учета в аксесс, но в связи с ростом организации и числа сотрудников  потребовался переход в ВЕБ.
Путем длительных поисков вышел на TOTUM

-  Очень порадовало наличие русской документации.
- Гибкость.
- Минимум кода.
- Практически моментальная связь с разработчиком в telegram (отдельно ОГРОМНАЯ благодарность за оперативность и помощь)
      Система работает больше года  и за это время прекрасно себя зарекомендовала

**Comment by ttm-alexeypolunin** [comment]
Спасибо за отзыв! Какое у вас количество пользователей если не секрет?

**Comment by ivev7** [comment]
Не секрет 37  , но это все , постоянно работающих примерно 8-10
